addons/website_event/models/website_visitor.py

```diff
@@ -43,7 +43,7 @@ def _compute_email_phone(self):
         self.flush()
 
         for visitor in self.filtered(lambda visitor: not visitor.email or not visitor.mobile):
-            linked_registrations = visitor.event_registration_ids.sorted(lambda reg: (reg.create_date, reg.id), reverse=False)
+            linked_registrations = visitor.sudo().event_registration_ids.sorted(lambda reg: (reg.create_date, reg.id), reverse=False)
```

Comment:
As it's yet another `sudo()` in `_compute_email_phone()`, don't you think it would be better to flag that compute method as `compute_sudo=True`?

Reply:
I was not sure we could make such a change to a field in stable. If this is ok then I'll do it (and remove the other added `sudo`s).

Reply:
I'am 99% sure it's ok. I asked confirmation just to be sure before r+.
Ty for the change